package.json

```diff
@@ -32,7 +32,7 @@
   "scripts": {
     "benchmarks": "ts-node-transpile-only test/benchmarks.ts",
     "build": "rm -rf lib && babel src --out-dir lib --source-maps inline --extensions '.ts'",
-    "lint": "eslint --ignore-path .gitignore --ext .js,.ts .",
+    "lint": "eslint --ignore-path .gitignore --ignore-pattern release.config.js --ext .js,.ts .",
```

Comment:
Question: why not lint the config? Does it break something?

Reply:
Breaks something with the tsconfig, the `--ignore-pattern` is a workaround. I'll take a look at fixing the tsconfig directly

```
0:0  error  Parsing error: "parserOptions.project" has been set for @typescript-eslint/parser.
The file does not match your project config: release.config.js.
The file must be included in at least one of the projects provided
```

Reply:
fixed!